test/AppNotificationTests/AppNotifications.Test.h

```diff
@@ -129,8 +129,9 @@ namespace AppNotifications::Test
             break;
 
         case ExpectedTransientProperties::DEFAULT:
+            // The default priority value is Default, and the default for SuppressDisplay is false.
             VERIFY_ARE_EQUAL(winrt::Microsoft::Windows::AppNotifications::AppNotificationPriority::Default, actual.Priority());
-            VERIFY_IS_TRUE(actual.SuppressDisplay());
+            VERIFY_IS_FALSE(actual.SuppressDisplay());
```

Comment:
Was an existing test failing because of this?

Reply:
No, because of the other test invoking the wrong BaseTestSuite function 😆. Once I fixed the naming thing, this verification step bug showed up.